src/python/detectors/hardcoded_ip_address/hardcoded_ip_address.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=hardcoded-ip-address@v1.0 defects=1}
+def hardcoded_ip_address_noncompliant():
+    from socket import AF_INET, SOCK_STREAM, socket
```

Comment:
could you make this import to the file level since it's used in both `noncompliant` and `compliant` method